addons/api/mirage/factories/host-catalog.js

```diff
@@ -31,8 +30,9 @@ let pluginTypeCounter = 1;
 export default factory.extend({
   id: () => generateId('hc_'),
 
-  // Cycle through available types
-  type: (i) => TYPES_HOST_CATALOG[i % TYPES_HOST_CATALOG.length],
+  // Generate on average 1 static catalog to 4 dynamic catalogs
```

Comment:
I had to change how we generate as we weren't getting any GCP plugins now without increasing the number of host catalogs in the scenario

Reply:
Nvm I undid this change as it turns out this behavior is load bearing in our tests